shared-libs/transitions/test/unit/transitions/registration.js

```diff
@@ -2798,7 +2800,9 @@ describe('registration', () => {
         transitionUtils.addRegistrationNotFoundError.callCount.should.equal(1);
         transitionUtils.addRegistrationNotFoundError.args[0].should.deep.equal([doc, registrationConfig]);
         contactTypeUtils.isPlace.callCount.should.equal(1);
-        contactTypeUtils.isPlace.args[0].should.deep.equal([{}, {_id: 'person', patient_id: '56987', type: 'person' }]);
+        contactTypeUtils.isPlace.args[0].should.deep.equal(
+          [{}, { _id: 'person', patient_id: '56987', type: 'person' }]
+        );
```

Comment:
Please add unit tests to cover this new functionality.

Reply:
Interesting there should be no change here. Think its formatting changes.

Reply:
Was failing the 120 line eslint length check on this test hence Formatted.

Reply:
I just picked this line to add the comment. I was referring to adding unit tests in general.

Reply:
Added tests for registration add_patient flow as well as function to get phone field.